### PR TITLE
Adapt ai recorder to vDAQ

### PR DESCRIPTION
**Pull request by ablot**
Solves #3 

- Works *only with sampling rate above 3kHz* 
- Not tested with NiDaq

@raacampbell fell free to make suggestions if you want me to organise things a bit or do any changes
I would also change the default firing rate to 3.5kHz to make sure it works out of the box for vDAQ but that's a detail

### CODE REVIEW

File: code/+sitools/@ai_recorder/ai_recorder.m [hunk 1]
```diff
@@ -183,16 +183,87 @@
...
+                
+                % * Set up analog inputs
+                for i = 1:numel(obj.AI_channels)
+                    obj.hTask.addChannel(obj.AI_channels(i));
+                end
+                
+                % * Configure the sampling rate and the size of the buffer in samples using the on-board sanple clock
+                obj.hTask.sampleRate = obj.sampleRate;
+                if obj.sampleRate < 3000
+                    fprintf('Warning! vDAQ seems weird with sampling rate below 3kHz./n')
```
Comment (raacampbell) [inline code/+sitools/@ai_recorder/ai_recorder.m]: wrong slash

File: code/+sitools/@ai_recorder/ai_recorder.m [hunk 2]
```diff
@@ -183,16 +183,87 @@
             % ai_recorder.connectToDAQ - Connect to the DAQ using the object properties
             %
             % Purpose
-            % Run this method to connect to an NI DAQ device for analog
+            % Run this method to connect to an DAQ device for analog
             % input using the parameters described in the properties of
             % this class. i.e.
-            % devName - the name of the NI device
+            % devName - the name of the device
```
Comment (raacampbell) [inline code/+sitools/@ai_recorder/ai_recorder.m:196]: Should probably put in the docs how to select vDAQ or NI DAQ? Or add examples in doc text? Is starting with a v a convention you set up? I guess it must be.

Comment (ablot) [reply code/+sitools/@ai_recorder/ai_recorder.m:196]: On my setup, the device name starts with v but it shouldn't matter as long as it matches what scanimage calls it.

File: code/+sitools/@ai_recorder/ai_recorder.m (hunk 1 above)
Comment (ablot) [reply code/+sitools/@ai_recorder/ai_recorder.m]: fixed

**Comment by raacampbell** [comment]
Let's just merge and carry on.

**Comment by ablot** [comment]
You might want to test it on a system with a Ni card before updating them :)

**Pull request merged**